src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -197,17 +176,23 @@ const NavHeader = (props) => {
                   {isFollow === undefined && isLoading ? (
                     <Shimmer width={normalize(100)} height={normalize(36)} />
                   ) : isFollow ? (
-                    <ButtonFollowing handleSetUnFollow={handleFollowTopic} />
+                    <ButtonFollowing
+                      handleSetUnFollow={onFollowButtonPress}
+                      followType={followType}
+                    />
                   ) : (
-                    <ButtonFollow handleSetFollow={handleFollowTopic} />
+                    <ButtonFollow handleSetFollow={onFollowButtonPress} />
                   )}
                 </Animated.View>
               </View>
             </View>
             <Animated.View
               style={[
                 styles.domain(isHeaderHide),
-                {bottom: normalize(isFollow ? 13 : 24), opacity: opacityHeaderAnimation}
+                {
+                  bottom: normalize(isFollow && followType === 'signed' ? 13 : 24),
+                  opacity: opacityHeaderAnimation
+                }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleSetUnFollow` prop of `ButtonFollowing` and `handleSetFollow` prop of `ButtonFollow` have been updated to use `onFollowButtonPress` instead of `handleFollowTopic`. Also, the bottom position of the domain is now dependent on both `isFollow` and `followType`. These changes seem fine as long as the `onFollowButtonPress` function is passed as a prop from the parent component and the `followType` prop is always provided and has a valid value.
```diff
- <ButtonFollowing handleSetUnFollow={handleFollowTopic} />
+ <ButtonFollowing handleSetUnFollow={onFollowButtonPress} followType={followType} />
- <ButtonFollow handleSetFollow={handleFollowTopic} />
+ <ButtonFollow handleSetFollow={onFollowButtonPress} />
- {bottom: normalize(isFollow ? 13 : 24), opacity: opacityHeaderAnimation}
+ {bottom: normalize(isFollow && followType === 'signed' ? 13 : 24), opacity: opacityHeaderAnimation}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->